biz.aQute.bndlib/src/aQute/bnd/osgi/resource/CapReq.java

```diff
@@ -20,26 +22,27 @@
 	private final Resource				resource;
 	private final Map<String,String>	directives;
 	private final Map<String,Object>	attributes;
+	private transient int				hashCode	= 0;
```

Comment:
Nothing. It is (a) notice to the programmer that the field hold a transient value (that can be recomputed) and (b) the value is not to be serialized if the class is Serializable. The latter does not apply here. I only marked it transient for the former reason.